File: java/IEC61970/Base/DC/DCGround.java
package IEC61970.Base.DC;

import IEC61970.Base.Domain.Inductance;
import IEC61970.Base.Domain.Resistance;

/**
 * A ground within a DC system.
 * @author selaost1
 * @version 1.0
 * @created 25-Dec-2023 8:31:55 PM
 */
public class DCGround extends DCConductingEquipment {

	/**
	 * Inductance to ground.
	 */
	public Inductance inductance;
	/**
	 * Resistance to ground.
	 */
	public Resistance r;

	public DCGround(){

	}
}//end DCGround

File: py/IEC61970/Base/DC/DCGround.py
from CIM_STD_PYTHON.TC57CIM.IEC61970.Base.DC import DCConductingEquipment
from CIM_STD_PYTHON.TC57CIM.IEC61970.Base.Domain.Inductance import Inductance
from CIM_STD_PYTHON.TC57CIM.IEC61970.Base.Domain.Resistance import Resistance


class DCGround(DCConductingEquipment):
    """
    A ground within a DC system.

    @author: selaost1
    @version: 1.0
    @created: 15-Dec-2023 4:38:27 PM
    """
    def __init__(self) -> None:
        """
        Default constructor
        """

        self.inductance: Inductance = Inductance()  # Inductance to ground
        self.r: Resistance = Resistance()  # Resistance to ground

